src/Compute/Compute/help/Add-AzVMSecret.md

```diff
@@ -27,7 +27,24 @@ For more information about the cmdlets, see [Azure Key Vault Cmdlets](/powershel
 
 ## EXAMPLES
 
-### Example 1: Add a secret to a virtual machine
+### Example 1: Add a secret to a virtual machine using the Azure Key Vault virtul machine extension
+To install certificates on a virtual machine it is recommended to use the [Azure Key Vault virtual machine extension for Linux](https://docs.microsoft.com/azure/virtual-machines/extensions/key-vault-linux) or the [Azure Key Vault virtual machine extension for Windows](https://docs.microsoft.com/azure/virtual-machines/extensions/key-vault-windows).
+
+```
+# Build settings
+    $settings = '{"secretsManagementSettings": 
+    { "pollingIntervalInS": "' + <pollingInterval> + 
+    '", "certificateStoreName": "' + <certStoreName> + 
+    '", "certificateStoreLocation": "' + <certStoreLoc> + 
+    '", "observedCertificates": ["' + <observedCert1> + '","' + <observedCert2> + '"] } }'
```

Comment:
You can define it PowerShell way then use `ConvertTo-Json`. It's more natural.

```suggestion
$settings = @{
    secretsManagementSettings = @{
        pollingIntervalInS       = "<pollingInterval>"
        certificateStoreName     = "<certStoreName>"
        certificateStoreLocation = "<certStoreLoc>"
        observedCertificates     = @("<observedCert1>", "<observedCert2>")
    } 
} | ConvertTo-Json
```

I'm not 100% sure if the result is as you expect so please give it a try